_transformations/scale/readme.md

```diff
@@ -14,17 +14,44 @@ install: auto
 
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges.
+The input string must be in numerical format.
```

Comment:
What about the output?

Reply:
The output is a simple string according to https://github.com/eclipse/smarthome/blob/master/extensions/transform/org.eclipse.smarthome.transform.scale/src/main/java/org/eclipse/smarthome/transform/scale/internal/ScaleTransformationService.java

Should we mention that?